web_app/calendar/html_template.py

```diff
@@ -138,3 +153,9 @@
                    year_2019_link=year_2019_link,
                    year_2020_link=year_2020_link)
 
+# ================ DAILY ========================
+
+task_cell = '''
```

Comment:
Why did you create this task_cell?
It looks like you set only color here.

Reply:
What do you mean? I generate cells for every daily task here.